Update EPS limits protection documentation in the API configuration page (#6006)
wazuh/wazuh-documentation: source/user-manual/api/configuration.rst

+------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Sub-fields | Allowed values       | Default value | Description                                                                                                                                                                                                                                                                                                                                 |
+============+======================+===============+=============================================================================================================================================================================================================================================================================================================================================+

> | allow      | yes, true, no, false | true          | Allow uploading configurations with modified EPS limits through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files if the ``<limits><eps>`` section inside the :ref:`global tag <reference_ossec_global>` is different from the previous one.                                                          |

Reviewer: ```suggestion
| allow      | yes, true, no, false | true          | Allow uploading configurations with modified EPS limits through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files if the ``<limits><eps>`` section inside the :ref:`global tag <reference_ossec_global>` has changed.                                                                                 |
```
